### PR TITLE
uds: move uds on a per port basis

### PR DESCRIPTION
**Pull request by maryamtahhan**
Fixup CNDP to work with https://github.com/intel/afxdp-plugins-for-kubernetes/pull/81 --> this has been tested in a Kubeadm cluster with the AF_XDP DP

This will affect/potentially break the golang and rust APIs

### CODE REVIEW

File: examples/cndpfwd/parse-args.c [hunk 1]
```diff
@@ -75,12 +75,12 @@ process_callback(jcfg_info_t *j __cne_unused, void *_obj, void *arg, int idx)
                 if (f->test == UNKNOWN_TEST)
                     f->test = get_app_mode(f->opts.mode);
             }
-        } else if (!strncmp(obj.opt->name, UDS_PATH_TAG, nlen)) {
-            if (obj.opt->val.type == STRING_OPT_TYPE) {
-                f->xdp_uds = udsc_handshake(obj.opt->val.str);
-                if (f->xdp_uds == NULL)
-                    CNE_ERR_RET("UDS handshake failed %s\n", strerror(errno));
-            }
+            // } else if (!strncmp(obj.opt->name, UDS_PATH_TAG, nlen)) {
```
Comment (maryamtahhan) [inline examples/cndpfwd/parse-args.c:78]: cleanup code

**Comment by maryamtahhan** [comment]
created 2 new issues to track the rust and golang work

- https://github.com/CloudNativeDataPlane/cndp/issues/352
- https://github.com/CloudNativeDataPlane/cndp/issues/353

File: tools/jsonc_gen_lport_groups.sh [hunk 2]
```diff
@@ -221,6 +220,7 @@ function emit_jsonc_lport_groups
             "threads": ["fwd:$i"],
             "pmd": "net_af_xdp",
             "skb_mode": $CNDP_COPY_MODE,
+            "uds_path": "/tmp/cndp.sock",
```
Comment (maryamtahhan) [inline tools/jsonc_gen_lport_groups.sh:223]: need to fix this up.

**Review by KeithWiles — APPROVED** [review]

File: tools/jsonc_gen_lport_groups.sh (hunk 2 above)
Comment (maryamtahhan) [reply tools/jsonc_gen_lport_groups.sh:223]: This is not a straight forward change for lport-groups. updated known issues for now to unblock the PR

**Review by KeithWiles — APPROVED** [review]

**Pull request merged**